### PR TITLE
fix(core): propagate meta properties into final TestResult

### PR DESCRIPTION
**Pull request by matzuk**

### CODE REVIEW

File: core/src/main/kotlin/com/malinskiy/marathon/execution/progress/PoolProgressAccumulator.kt [hunk 1]
```diff
@@ -19,11 +19,11 @@ import kotlin.math.roundToInt
 
 class PoolProgressAccumulator(
     private val poolId: DevicePoolId,
-    shard: TestShard,
-    private val configuration: Configuration,
+    val shard: TestShard,
+    val configuration: Configuration,
```
Comment (Malinskiy) [inline core/src/main/kotlin/com/malinskiy/marathon/execution/progress/PoolProgressAccumulator.kt]: why the change to public?

**Review by Malinskiy — CHANGES_REQUESTED** [review]

File: core/src/main/kotlin/com/malinskiy/marathon/execution/progress/PoolProgressAccumulator.kt (hunk 1 above)
Comment (matzuk) [reply core/src/main/kotlin/com/malinskiy/marathon/execution/progress/PoolProgressAccumulator.kt]: fixed

Comment (Malinskiy) [reply core/src/main/kotlin/com/malinskiy/marathon/execution/progress/PoolProgressAccumulator.kt]: Where? I still see `val configuration: Configuration,`

Comment (matzuk) [reply core/src/main/kotlin/com/malinskiy/marathon/execution/progress/PoolProgressAccumulator.kt]: Please check the file. You see the outdated information (GitHub specific).
